src/main/java/service/impl/FileWriterServiceImpl.java

```diff
@@ -0,0 +1,28 @@
+package service.impl;
+
+import db.Storage;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import service.FileWriterService;
+
+public class FileWriterServiceImpl implements FileWriterService {
```

Comment:
this class is responsible for 2 tasks: form report from storage  and write ready report to file. Let's leave here only writing so we can reuse this class if needed elsewhere